board/piksiv3/post_image.sh

```diff
@@ -113,6 +121,12 @@ generate_prod()
   # Ensure log is empty
   true >$LOG_FILE
 
+  $UBOOT_DEV_DIR/tools/mkimage                                                \
+  -A arm -T firmware -C none -O u-boot -n "FPGA bitstream"                    \
+  -d $FIRMWARE_DIR/piksi_fpga.bit                                             \
+  $OUTPUT_DIR/piksi_fpga.img                                                  \
+    | tee -a $LOG_FILE &>$LOG_DEBUG
+
```

Comment:
This seems like a questionable to me ... why does generate_prod() not need to mkimage generate a piksi_fpga.img?